modules/site-health/admin-site-health.php

```diff
@@ -83,175 +83,24 @@ protected function exclude_language_keys() {
 		);
 	}
 
-	/**
-	 * Formats an array to display in options information.
-	 *
-	 * @since 2.8
-	 *
-	 * @param array $array An array of formatted data.
-	 * @return string
-	 */
-	protected function format_array( $array ) {
-		array_walk(
-			$array,
-			function ( &$value, $key ) {
-				if ( is_array( $value ) ) {
-					$ids = implode( ' , ', $value );
-					$value = "$key => $ids";
-				} else {
-					$value = "$key => $value";
-				}
-			}
-		);
-		return implode( ' | ', $array );
-	}
```

Comment:
Instead of moving this method to `Abstract_Option`,  you could keep it here make it public and static. That way you could keep using it here and in option classes too.

Reply:
I agree but... what is the benefit?
It was there, @Hug0-Drelon asked me to move it, now you're asking me to go back and do differently... Okay but why ? :/ (just to understand what you're asking me)